airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -160,7 +163,23 @@ def convert_schema_into_query(
         return query_template
 
     @staticmethod
-    def get_field_value(field_value: GoogleAdsRow, field: str, schema_type: Mapping[str, Any]) -> str:
+    def serialize_protobuf_message(message: Message) -> str:
+        """
+        This method tryies to serialize a given object into JSON if the given object is a protocol bugger message.
+        If not, the normal str() is applied. That helps keep the complex objects written as parsable structures.
+        """
+        try:
+            # The code is accessing "private" _pb value, because the message itself does not
+            # contain public PB attribute and for google ads message-based classes, like AdTextAd,
+            # that's apparently the only way to get the actual protobuf message that cab be serialzied
+            # into json.
+            return json_format.MessageToJson(message._pb, indent=0).replace('\n', '')
```

Comment:
Can we add `# type: ignore[union-attr] # if _pb does not exist then we fall back to using the string representation` . We plan to start using mypy to enforce type safety and i have a feeling accessing a potentially non-existent field will cause it to fail

Reply:
the mypy convention over ignore the type check should actually go on the line of code what has the type mismatch, so it should look like:

```
return json_format.MessageToJson(message._pb, indent=0).replace('\n', '')  # type: ignore[union-attr] # add explanation for why here
```